os/board/rtl8720e/src/component/filesystem/ftl/ftl.c

```diff
@@ -1440,6 +1440,10 @@ void ftl_mapping_table_init(void)
 		//                                  MAPPING_TABLE_SIZE);//table is initialised as 0
 
 		ftl_mapping_table = rtw_zmalloc(MAPPING_TABLE_SIZE);
+		if (NULL == ftl_mapping_table) {
+			FTL_PRINTF(FTL_LEVEL_ERROR, "[ftl] ftl_mapping_table rtw_zmalloc fail!");
+			return;
```

Comment:
This will cause uncomplete init. But ftl layer will execute next functions without init. Is it ok?

Reply:
Is it ok if FTL_ASSERT is added after FTL_PRINTF at line 1444, such that if init is incomplete it will stop execution.

Reply:
If it's mandatory, then assert will be ok. If ftl is working without init, then, current is ok. It depends on the ftl structure.

Reply:
@yeetee179 Could you let me know how will you do for this? Sorry but this is urgent for me.

Reply:
I will add the assertion here, I will commit and push again.